### PR TITLE
Unit test prep

### PR DESCRIPTION
**Pull request by Avitus27**

### CODE REVIEW

**Comment by Avitus27** [comment]
Python analysis passed in LGTM

File: flask-bot/unit_tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import sys
+import os
+
+sys.path.append(os.path.realpath(os.path.dirname(__file__) + "/.."))
```
Comment (Sherkdavid) [inline flask-bot/unit_tests/__init__.py]: I'm a fan of pathlib for operations like this, nothing that would block this going in but maybe in future

**Review by Sherkdavid — APPROVED** [review]

File: flask-bot/unit_tests/__init__.py (hunk 1 above)
Comment (Avitus27) [reply flask-bot/unit_tests/__init__.py]: Taking your advice and updating this now, next change will resolve this!
You're right, it looks a lot cleaner :P

**Comment by Avitus27** [comment]
Python analysis passed, merging

**Pull request merged**